source/common/config/well_known_names.h

```diff
@@ -47,6 +47,8 @@ class MetadataEnvoyLbKeyValues {
   const std::string CANARY = "canary";
   // Key in envoy.lb filter namespace for the key to use to hash an endpoint.
   const std::string HASH_KEY = "hash_key";
+  // Key in envoy.lb filter namespace for providing fallback metadata
+  const std::string FALLBACK_LIST = "fallback_list";
```

Comment:
Probably should note that this key appears in the RouteAction metadata_match field and has no special meaning elsewhere. (The other two appear in endpoint metadata.)

Reply:
It might also be prudent to add a runtime guard around this feature in the unlikely event that someone happens to have metadata with this key and is using the subset load balancer to match on it.

Reply:
> It might also be prudent to add a runtime guard around this feature in the unlikely event that someone happens to have metadata with this key

`fallback_list` has no special meaning, if the user doesn't explicitly enable `metadata_fallback_policy: FALLBACK_LIST` in config. 
Do you mean we should take another precaution and check if endpoints don't have this metadata key set when user enabled `metadata_fallback_policy: FALLBACK_LIST`?

Reply:
> fallback_list has no special meaning (and can be used as a regular metadata key), if user doesn't explicitly enable metadata_fallback_policy: FALLBACK_LIST in config.

Ok, as long as the keys isn't used unless explicitly configured, that's probably fine.

> LMK if this is what you have in mind

I meant a comment on the constant about where it gets used.